### DISCUSSION TITLE
Quarto install on linux rhel7 server -- command not found

### DISCUSSION THREAD

**Discussion by maxwellkonnaris** [quarto-dev/quarto-cli#4074] [Q&A] [answered]
Hi, Im trying to install quarto on linux rhel7 server:

commands : 

wget https://github.com/quarto-dev/quarto-cli/releases/download/v1.2.313/quarto-1.2.313-linux-rhel7-amd64.tar.gz

tar -xvzf quarto-1.2.313-linux-rhel7-amd64.tar.gz

bash: quarto: command not found

I tried quarto.exe and quarto.cmd, both dont work

Do I have to place the path quarto somewhere like in .bashrc? suprised theres no .install file when you unzip the tar. Anyone deal with this / know what to do so I can use quarto? When I press render in Vscode it says I need to download quarto CLI so its definitely not recognizing. also if I type in quarto from terminal its not recognizing as well. 

Thanks!

**Comment by cderv** [comment, marked as answer]
I believe with the `.tar.gz` you then need to create symlink to some place on your PATH, or add the unzipp directory to PATH. 

otherwise, the CLI won't be available. 

@dragonstyle should we maybe add a INSTALL file or something in the bundle ? Do we have a installation page somewhere ? I think we still need to add it in our doc.

**Reply by dragonstyle** [reply]
Yes we should do a better job of this. For our AMD tarball installation, we have the following:

https://quarto.org/docs/download/tarball.html?version=1.2.313&idPrefix=download

The specific commands provided are not RHEL specific, but the general outline of steps is accurate and could be used if you're familiar with configuring your environment.

It would be great to improve our documentation for RHEL to include the same type of instructions...

**Reply by cderv** [reply]
I have opened https://github.com/quarto-dev/quarto-web/issues/585 to track this
